common/oe_host_string.h

```diff
@@ -1,6 +1,14 @@
 // Copyright (c) Open Enclave SDK contributors.
 // Licensed under the MIT License.
 
+/*
+ * There are separate OE functions for strerror and strerror_s.
+ * Therefore we suppress the warning about not using more secure CRT functions.
+ */
+#ifdef _WIN32
+#pragma warning(disable : 4996)
```

Comment:
rather than disabling it all together, it's better to wrap the relevant parts (including this line) in
#pragma warning(push)
#pragma warning(pop)